extensions/amp-accordion/1.0/accordion.js

```diff
@@ -253,14 +267,22 @@ export function AccordionSection({
     prefix,
   } = useContext(AccordionContext);
 
+  const expanded = isExpanded ? isExpanded(id, defaultExpanded) : expandedState;
+  const animate = contextAnimate ?? defaultAnimate;
+  const contentId = `${prefix || 'a'}-content-${id}-${suffix}`;
+
   useEffect(() => {
     hasMountedRef.current = true;
     return () => (hasMountedRef.current = false);
   }, []);
 
+  const sectionPropsRef = useRef(
```

Comment:
Nit, but since we simplified, let's rename to `onExpandStateChangeRef` and add a comment as of why we are storing a prop as a ref, since it's not very common.

Reply:
Got it, updated in the latest revision!